packages/searchbase/examples/with-react/src/index.js

```diff
@@ -15,7 +15,10 @@ class App extends React.Component {
     this.searchBase = new SearchBase({
       index,
       url,
-      credentials
+      credentials,
+      appbaseConfig: {
+        enableTelemetry: false
```

Comment:
Why are you setting the `enableTelemetry` property to `false`?
There is no need to set a default value of `appbaseConfig`.